src/main/docker/redis-cluster.yml

```diff
@@ -0,0 +1,72 @@
+version: '2'
+services:
+  order-redis:
+    image: redis:6.0.4
```

Comment:
version 6.4 available. 6.0 is 2 years old
https://docs.redis.com/latest/rs/release-notes/